static/js/TextsPage.jsx

```diff
@@ -108,6 +109,7 @@ const TextsPage = ({categories, settings, setCategories, onCompareBack, openSear
             { about }
             { dedication }
             { libraryMessage }
+            { !multiPanel && <RecentlyViewed toggleSignUpModal={toggleSignUpModal} mobile={true}/>}
```

Comment:
I could implement another component RecentlyViewedMobile but it would require taking 2 functions out of RecentlyViewed component and having them be standalone functions that are shared.  Is this preferable over the current approach?